### DISCUSSION TITLE
What is the difference between “Quick Save” and  "Auto Finish"

### DISCUSSION THREAD

**Discussion by DearZack** [localsend/localsend#1532] [Discussion (English, International)] [answered]
What is the difference between “Quick Save” and  "Auto Finish"

**Comment by Tienisto** [comment, marked as answer]
Quick Save: Automatically accept incoming requests without having to tap "Accept"
Auto Finish: Automatically close the session when it was completed. Normally, you have to tap "Finish" to be able to accept further requests.
